lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -784,7 +796,8 @@ export abstract class ClientApplication {
             return null;
         }
 
-        const account = this.browserStorage.getAccountInfoFilteredBy({username});
+        const account: AccountInfo | null = this.browserStorage.getAccountInfoFilteredBy({username}, this.config.auth.multiTenantAccountsEnabled);
```

Comment:
Can we also add `@deprecated - Use getAccountsByFilter instead` to each of the doc comments over the old getAccountBy APIs?

Reply:
Done